tests/UnitTests/CoreGraphics.drawing/EntryPoint.cpp

```diff
@@ -54,7 +48,22 @@ class CommandLineDrawingTestConfigImpl : public DrawingTestConfigImpl {
 
 public:
     CommandLineDrawingTestConfigImpl(int argc, char** argv)
```

Comment:
like: there are so many ifdefs in this file it might simply be easier to have a win32 file that has its own config impl and the os x file that keeps the commandline one.